src/System.Threading.Thread/src/System/Threading/Thread.cs

```diff
@@ -209,12 +305,14 @@ public void Resume()
         public static void BeginThreadAffinity() { }
         public static void EndThreadAffinity() { }
 
+#if !NETNATIVE
```

Comment:
Unless I missed something, these don't depend on any runtime support, they are fully implemented here. It seems like it shouldn't be necessary to disable them.